src/main/connector/system/System/System.yaml

```diff
@@ -119,11 +119,26 @@ metrics:
     description: Total number of processes created over uptime of the host
     type: Counter
     unit: "{process}"
+  system.process.pid:
+    description: Process ID
+    type: Gauge
+  system.process.cpu_utilization:
+    description: Cpu utilization of the process

```

Comment:
`CPU` in all caps

Reply:
Should probably be `system.process.cpu.utilization`.
Also, can't we have the CPU time as a counter of seconds, like `system.process.cpu.time`?